mobly/controllers/android_device.py

```diff
@@ -1000,15 +1000,14 @@ def take_screenshot(self, destination, filename=None):
 
     Args:
       destination: string, full path to the directory to save in.
-      filename: string, additional name of the screenshot will be added after the
-        file_type:screenshot if given.
+      filename: string, additional name of the screenshot will be added if given.
```

Comment:
How about we simply allow users to pass in `prefix`.
And the default value of `prefix` is "screenshot"?